src/styles/main.scss

```
@@ -1,3 +1,222 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
 body {
+  font-family: inherit;
+  font-style: inherit;
+}
+
+$padding-header: 0 50px;
+
+$font-small: 400;
+$font-medium: 500;
+$font-bold: 700;
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: $padding-header;
+  margin-bottom: 50px;
+  
+  font-size: 12px;
+  font-weight: $font-medium;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+$logo-size: 40px;
+
+.logo {
+  display: flex;
+  align-self: center;
+  height: $logo-size;
+  width: $logo-size;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav__item {
+  display: flex;
+  padding: 0;
   margin: 0;
 }
+
+.nav__link {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+
+  text-transform: uppercase;
+  text-decoration: none;
+  color: inherit;
+}
```

используй вложенность
```suggestion
.nav {
  display: flex;
  align-items: center;

 &__item {
  display: flex;
  padding: 0;
  margin: 0;
}

&__link {
  display: flex;
  justify-content: space-between;
  align-items: center;
  height: 60px;

    text-transform: uppercase;
    text-decoration: none;
    color: inherit;
  }
}
```